src/validations.json

```diff
@@ -0,0 +1,3 @@
+{
+    "do_not_delete": []
```

Comment:
Probably put the sagemaker endpoint in here

Reply:
I did and it caused problems on updating the model's values. Will do some more testing.